### DISCUSSION TITLE
Unable to use percy cli 1.1

### DISCUSSION THREAD

**Discussion by gluip** [percy/cli#903] [Support] [answered]
I am unable to install Percy cli on windows (Node 14.17.6 and npm 6.14.15). Percy CLI version 1.1.0

1. create new folder (percytest)
2. npm init in this folder (all defaults)
3. npm install --save-dev @percy/cli (this installs @percy/cli 1.1)
4. percy (in same command window)
5. error:
**'percy' is not recognized as an internal or external command,
operable program or batch file.**

**Comment by Robdel12** [comment, marked as answer]
Hey @gluip! This is because when it’s installed locally in a projects dependencies, the binary isn’t added to the windows equivalent of `$PATH`. You will need to use `npx` (`npx percy …`) or call the binary directly (`.\node_modules\.bin\percy …`). `npx` does the job of looking for a binary that matches the name for you.

**Reply by gluip** [reply]
thanks!
